hidapi/hidapi.h

```diff
@@ -78,6 +78,30 @@ extern "C" {
 		struct hid_device_;
 		typedef struct hid_device_ hid_device; /**< opaque hidapi structure */
 
+		/** @brief HID underlying bus types.
+
+			@ingroup API
+		*/
+		typedef enum {
+			/* Universal Serial Bus */
+			HID_BUS_USB = 0x01,
```

Comment:
we've already faced a name conflict with `hid_init` (on some of the BSD platforms);
let's prefix it as `HID_API_` or `HIDAPI_` instead of just `HID_` to decrease the chance of name conflict

Reply:
done